src/main/java/no/unit/nva/cristin/projects/CristinApiClient.java

```diff
@@ -75,21 +67,24 @@ protected URL generateQueryProjectsUrl(Map<String, String> parameters) throws Ma
 
     protected URL generateGetProjectUrl(String id, String language) throws MalformedURLException, URISyntaxException {
         URI uri = new URIBuilder()
-                .setScheme(HTTPS)
-                .setHost(CRISTIN_API_HOST)
-                .setPath(CRISTIN_API_PROJECTS_PATH + id)
-                .addParameter("lang", language)
-                .build();
+            .setScheme(HTTPS)
+            .setHost(CRISTIN_API_HOST)
+            .setPath(CRISTIN_API_PROJECTS_PATH + id)
+            .addParameter("lang", language)
+            .build();
         return uri.toURL();
     }
 
-    protected static <T> T fromJson(InputStreamReader reader, Class<T> classOfT) throws IOException {
-        try {
-            return new Gson().fromJson(reader, classOfT);
-        } catch (JsonSyntaxException e) {
-            final String s = e.getMessage() + " " + reader;
-            throw new IOException(s, e);
-        }
+    private List<Project> enrichProjects(String language, List<Project> projects) {
```

Comment:
It's a bit weird that we consume a list of projects to produce a list of projects

Reply:
it was there from before, i just extracted it into new method.